src/components/data/DataTable.tsx

```diff
@@ -40,6 +40,9 @@ type ComponentProps<Row> = {
   /** Callback to render an individual table cell */
   renderItem?: (r: Row, field: keyof Row) => ComponentChildren;
   title: string;
+
+  /** Turn off outer table borders */
+  borderless?: boolean;
```

Comment:
I was confused that this prop was only added in the props type and no executable code changed. I realized that this works because this prop is captured by `htmlAttributes`. That rest property name is now misleading since we're including `Table` props too. Perhaps we _should_ continue to pass this prop and any forwarded non-HTML props explicitly to avoid confusion.

Reply:
Yep, makes sense.